src/gilded_rose.js

```diff
@@ -74,5 +76,6 @@ export function updateQuality(items) {
     }
     // handles decrementing of sell in for all items except sulfuras
     items[i].sell_in = mutateItemSellIn(items[i].sell_in);
+    return result
   }
```

Comment:
I think this needs to move one bracket down (between 80-81). Thank you for sharing your progress on this Flash, I know that this card was not an easy one!